### DISCUSSION TITLE
What's the WebDAV path for an album?

### DISCUSSION THREAD

**Discussion by averymd** [photoprism/photoprism#4585] [Q&A] [answered]
I want to be able to easily keep a collection of gifs on my phone that are sourced from an album in Photoprism. My plan was to use FolderSync to just pull the gifs from Photoprism into a folder on my phone using WebDAV.

The problem is that the gifs are not in a specific _folder_ among all my photos, they're in a Photoprism _album_, so the `/originals/` path given in the WebDAV-related documentation isn't the best way to get at them. 

I tried seeing if I could use the view or share links for an album as a WebDAV path, or the URL when downloading an album. None of those seem to be wired up for WebDAV calls. 

Is there a way to do this?

**Comment by graciousgrey** [comment, marked as answer]
As albums are virtual and not reflected in the filesystem, there is no path for them in originals.

It is possible to share albums from PhotoPrism with other WebDAV servers as described here:
- https://docs.photoprism.app/user-guide/settings/sync/#add-service
- https://docs.photoprism.app/user-guide/share/services-share/#share-albums-with-other-apps

However I am not sure if foldersync on your phone can be used for this.

**Reply by averymd** [reply]
Thank you for the reply! I'm going to go ahead and move those files into a single folder for simplicity.
